addOns/paramminer/src/main/javahelp/org/zaproxy/addon/paramminer/resources/help_ar_SA/contents/about.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Simple Example - About
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Simple Example - About</H1>
+
+<H2>Source Code</H2>
+<a href="https://github.com/zaproxy/zap-extensions/tree/main/addOns/simpleexample">https://github.com/zaproxy/zap-extensions/tree/main/addOns/simpleexample</a>
+
+<H2>Authors</H2>
+ZAP Dev Team
+
+<H2>History</H2>
+
+<H3>Version 2</H3>
+Updated to include example 'about' page.
+
+<H3>Version 1</H3>
+First Version
```

Comment:
You can drop the help folders other than `help`. The other locales will be added/updated over time via crowdin.